docs/configuration.md

```diff
@@ -2,20 +2,41 @@
 
 This document provides an overview of configuration settings for increasing hub performance.
 
+1. [Culler settings](#culler)
+   1. [Frequency](#culler-frequency)
+   2. [Concurrency limit](#culler-concurrency)
+   3. [Timeout](#culler-timeout)
+   4. [Notebook culler](#notebook-culler)
+2. [Activity intervals](#activity)
+   1. [`activity_resolution`](#activity-resolution)
+   2. [`last_activity_interval`](#last-activity-interval)
+   3. [`JUPYERHUB_ACTIVITY_INTERVAL`](#hub-activity-interval)
+3. [Startup time](#startup)
+   1. [`init_spawners_timeout`](#spawners-timeout)
+4. [Other settings](#other)
+   1. [`k8s_threadpool_api_workers`](#kubespawner-thread)
+   2. [Disable events](#kubespawner-events)
+5. [References](#references)
+
+
+<a name="culler"></a>
```

Comment:
nit: each of the sections creates an implicit link so we don't really need the explicit anchors like this, i.e. this is just the `#culler-settings` link. I'm not sure if you wanted explicit links though. I guess one advantage of explicit links is the section title can change and the link isn't broken.